### PR TITLE
chore(docs): medium articles 2022

### PR DESCRIPTION
**Pull request by sstraatemans**
adding content: medium articles 2022

### CODE REVIEW

File: packages/apps/docs/src/pages/docs/blogchain/2022/calling-web2-builders-kadena-eases-transition-to-web3-2022-03-09.mdx [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+and other community members. We attribute this growth to Kadena’s many technical
+advantages:
+
+1.  Mainstream accessibility
+
+2.  True decentralization
+
+3.  Infinitely scalable
+
+4.  Energy efficient\*
```
Comment (realdreamer) [inline packages/apps/docs/src/pages/docs/blogchain/2022/calling-web2-builders-kadena-eases-transition-to-web3-2022-03-09.mdx:144]: `\*` is it intended?

File: packages/apps/docs/src/pages/docs/blogchain/2022/deprecation-notice-for-module-guards-and-pact-guards-2022-10-03.mdx [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+Fortunately, building user guards using require-capability offers a truly secure
+mitigation. Capabilities now secure and scope guards alongside critical module
+code. All of the features that make Pact capabilities sound are now extended to
+guards for robust autonomous resource ownership.
+
+### Capability User Guards
+
+Capability User Guards are a design pattern for safe autonomous guards in Pact.
+They are composed of three code elements: a **capability,** a **predicate
+function,** and a **user guard.** The latter can be offered as a “constructor
```
Comment (realdreamer) [inline packages/apps/docs/src/pages/docs/blogchain/2022/deprecation-notice-for-module-guards-and-pact-guards-2022-10-03.mdx:73]:
```suggestion
They are composed of three code elements: a **capability**, a **predicate
function**, and a **user guard**. The latter can be offered as a “constructor
```

File: packages/apps/docs/src/pages/docs/blogchain/2022/deprecation-notice-for-module-guards-and-pact-guards-2022-10-03.mdx [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+
+This, of course, meant that module code invoking the guard-handling code had to
+be protected by other predicates and capabilities. As such, the code could have
+bugs as there was no explicit authorization, or scoping of said authorization.
+Capability-based guards solve these potential issues.
+
+However, the later introduction of module references now brings third-party code
+into the mix, with the unfortunate property that **anywhere the owning module
+calls into module reference code, that third-party code now passes the module
+guard.** This is also true for defpact guards: while the current Pact ID is
```
Comment (realdreamer) [inline packages/apps/docs/src/pages/docs/blogchain/2022/deprecation-notice-for-module-guards-and-pact-guards-2022-10-03.mdx:61]:
```suggestion
guard**. This is also true for defpact guards: while the current Pact ID is
```

**Review by realdreamer — APPROVED** [review]

**Review by barthuijgen — APPROVED** [review]

File: packages/apps/docs/src/pages/docs/blogchain/2022/calling-web2-builders-kadena-eases-transition-to-web3-2022-03-09.mdx (hunk 1 above)
Comment (sstraatemans) [reply packages/apps/docs/src/pages/docs/blogchain/2022/calling-web2-builders-kadena-eases-transition-to-web3-2022-03-09.mdx:144]: removed

**Pull request merged**